src/screens/Profile/FavoriteList/SortFavorites.tsx

```diff
@@ -0,0 +1,122 @@
+import {CompositeNavigationProp} from '@react-navigation/native';
+import {StackNavigationProp} from '@react-navigation/stack';
+import React, {useState} from 'react';
+import {View} from 'react-native';
+import {PanGestureHandler} from 'react-native-gesture-handler';
+import Animated from 'react-native-reanimated';
+import {SafeAreaView} from 'react-native-safe-area-context';
+import {ProfileStackParams} from '..';
+import {Close, Confirm} from '../../../assets/svg/icons/actions';
+import SvgDragHandle from '../../../assets/svg/icons/actions/DragHandle';
+import Button, {ButtonGroup} from '../../../components/button';
+import List from '../../../components/item-groups';
+import ThemeIcon from '../../../components/theme-icon';
+import {useFavorites} from '../../../favorites/FavoritesContext';
+import MessageBox from '../../../message-box';
+import {TabNavigatorParams} from '../../../navigation/TabNavigator';
+import {StyleSheet, Theme} from '../../../theme';
+import insets from '../../../utils/insets';
+import BackHeader from '../BackHeader';
+import {SortableList} from './SortableList';
+
+export type ProfileScreenNavigationProp = StackNavigationProp<
+  ProfileStackParams,
+  'ProfileHome'
+>;
+
+type ProfileNearbyScreenNavigationProp = CompositeNavigationProp<
+  StackNavigationProp<TabNavigatorParams, 'Profile'>,
+  ProfileScreenNavigationProp
+>;
+
+type ProfileScreenProps = {
+  navigation: ProfileNearbyScreenNavigationProp;
+};
+
+export default function SortableFavoriteList({navigation}: ProfileScreenProps) {
+  const style = useProfileStyle();
+  const {favorites, setFavorites} = useFavorites();
+  const items = favorites ?? [];
+  const [sortedItems, setSortedItems] = useState(items);
+  const [error, setError] = useState<string | null>(null);
+
+  const saveOrder = async () => {
+    try {
+      await setFavorites(sortedItems);
+      navigation.goBack();
+    } catch (_) {
+      setError(
+        'Ooops. Fikk ikke til å lagre favoritter. Prøv igjen er du snill. 🤞',
+      );
+    }
+  };
+
+  return (
+    <SafeAreaView style={style.container}>
+      <BackHeader title="Endre rekkefølge" closeIcon />
+
+      {error && (
+        <MessageBox type="error" message={error} containerStyle={style.error} />
+      )}
+
+      <SortableList
```

Comment:
A11y: We'll need to provide an alternative to dragging for sorting these items, so that they're accessible also when navigating with assistive technologies.

Reply:
Yes. I've been thinking of this also, but unsure how to do this best now. Might want to introduce some accessibility helper buttons for moving, but I think maybe we can look into this later?